### PR TITLE
GA events

### PR DESCRIPTION
**Pull request by jchris**
This is an initial pass at https://github.com/fauna/dashboard/issues/123

### CODE REVIEW

File: src/schema/components/class-delete.js [hunk 1]
```diff
@@ -9,11 +10,14 @@ import { faunaClient } from "../../authentication"
 import { buildResourceUrl } from "../../router"
 
 export const ClassDelete = ({ client, path, dbUrl, clazz }) => {
-  const onDelete = () => notify("Class deleted successfully", dispatch =>
-    dispatch(deleteClass(client, path, clazz.get("name"))).then(() =>
-      browserHistory.push(buildResourceUrl(dbUrl, "classes"))
+  const onDelete = () => {
+    ReactGA.event({category: "schema", action: "class-create"});
```
Comment (erickpintor) [inline src/schema/components/class-delete.js]: Shouldn't this be `class-delete` instead?

Comment (jchris) [inline src/schema/components/class-delete.js]: thanks!

File: src/notifications/notification-center.js [hunk 2]
```diff
@@ -15,6 +16,7 @@ const Actions = {
 
 export const pushNotification = (type, message) => (dispatch) => {
   const notification = Map.of("type", type, "message", message)
+  ReactGA.event({category: "notify", action: type, label: message});
```
Comment (erickpintor) [inline src/notifications/notification-center.js]: This one is still bugging me but I can't think about a different place if you really want to track all notifications.

Comment (jchris) [inline src/notifications/notification-center.js]: if it's too much i'll turn it off, first want to see what GA makes of it

**Pull request merged**